tests/src/Functional/DeveloperTest.php

```diff
@@ -69,12 +95,35 @@ protected function setUp() {
     $user_settings = $this->config('user.settings');
     $user_settings->set('register', USER_REGISTER_VISITORS_ADMINISTRATIVE_APPROVAL)->save(TRUE);
     $this->developerStorage = $this->container->get('entity_type.manager')->getStorage('developer');
+    $this->sdkConnector = $this->container->get('apigee_edge.sdk_connector');
+
+    $name = strtolower($this->randomMachineName());
+    $this->developer = Developer::create([
```

Comment:
I'd rather start using the _injected_ entity type manager service for CRUDL operations, instead of these static methods on the entity object. You have already acquired the developer storage above from the entity type manager. Use that object in this test to CRUDL developers.